ibis/backends/tests/test_array.py

```diff
@@ -1478,3 +1471,104 @@ def test_table_unnest_array_of_struct_of_array(con):
     result = con.execute(expr)
     expected = pd.DataFrame({"x": [1, 1, 1, 2, 2, 2, 3, 3, 3]})
     tm.assert_frame_equal(result, expected)
+
+
+notimpl_aggs = pytest.mark.notimpl(
+    ["datafusion", "flink", "pandas", "dask"], raises=com.OperationNotDefinedError
+)
+
+
+def _agg_with_nulls(agg, x):
+    if x is None:
+        return None
+    x = [y for y in x if not pd.isna(y)]
+    if not x:
+        return None
+    return agg(x)
+
+
+@pytest.mark.parametrize(
+    ("agg", "baseline_func"),
+    [
+        (ir.ArrayValue.sums, partial(_agg_with_nulls, sum)),
```

Comment:
Nit: I personally find partial mix more confusing than just writing out the lambda so I almost always do that. Not a blocker for sure, but if you ever aren't sure which to use, there is a little weight that could tip the scales for you